src/app/components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component.ts

```diff
@@ -209,6 +209,15 @@ export class ChallengeviewallsubmissionsComponent implements OnInit, AfterViewIn
     this.apiService.getUrl(API_PATH).subscribe(
       data => {
         SELF.submissions = data['results'];
+        for (let i = 0; i < SELF.submissions.length; i++) {
+          if (SELF.submissions[i].is_public) {
```

Comment:
Use the ternary operator to simplify these statements.

Reply:
@lunayach done